src/modules/esl-utils/decorators/attr.ts

```diff
@@ -15,6 +15,13 @@ type AttrDescriptor<T = string> = {
   name?: string;
   /** Create getter only */
   readonly?: boolean;
+  /** Find value to inherit across closest elements in DOM tree based on declared attribute name (in case of string format)
+   * or same attribute name of current element (boolean value).
+   * Example, attribute 'ignore' with configuration:
+   * inherit: 'alt-ignore' - searches ignore or data-ignore attr (in case of dataAttr: true) on this element or alt-ignore attribute on closest parent
+   * inherit: true - searches ignore or data-ignore attr (in case of dataAttr: true) on this element or on closest parent
+   */
```

Comment:
Just another interpretation, but still not clear enough. 
The main point - let's think about the description to make it more clear. 
```suggestion
  /** 
   * Inherits value from the attribute of the parent elements in the DOM tree. 
   * Uses the same attribute name as used for its own attribute if declared as `inherit: true`.
   * E.g., `@attr({inherit: true}) ignore;` - will search for `ignore` attribute on parents if its own value is not declared,
   * while `@attr({inherit: true, dataAttr}) ignore;` will look for `data-ignore` own or parent's attribute.
   * If the `inherit` value is declared as a string, then it considers the `inherit` parameter value as the closest attribute name to find the value to inherit.  
   * For example:
   * `@attr({inherit: 'alt-ignore'}) ignore;` - searches for its own `ignore` attribute (or 'data-ignore' in case of dataAttr: true) or consumes `alt-ignore` attribute (own or from the closest parent).
   */
```

Reply:
```suggestion
  /**
   * Specifies the attribute inheritance behavior. If `inherit` is set to `true`, the attribute will inherit the value from the same-named attribute of the closest parent element in the DOM tree. 
   * For instance, `@attr({inherit: true}) ignore;` will look for an `ignore` attribute in the parent elements if it's not defined in the current element. 
   * If `dataAttr` is also true, it will search for `data-ignore` instead.
   * 
   * If `inherit` is set to a string, it will use this string as the attribute name to search for in the parent elements. 
   * For example, `@attr({inherit: 'alt-ignore'}) ignore;` will first look for its own `ignore` attribute (or 'data-ignore' if `dataAttr` is true), and if not found, it will look for an `alt-ignore` attribute in the parent elements.
   */
```